examples/backtest.ipynb

```diff
@@ -23,15 +23,33 @@
     "\n",
```

Comment:
<p>If you decide to use trend as feature add the appropriate name for the output column, now it is set with repr</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>